src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -274,10 +272,19 @@ public static Vector128<T> Zero
                     AdvSimd.MultiplyWideningLower(
                         a.GetLower().AsUInt32(), b.GetLower().AsUInt32()).AsUInt16(),
                     AdvSimd.MultiplyWideningUpper(
-                        a.AsUInt32(),
-                        b.AsUInt32()).AsUInt16()).AsUInt64().As<ulong, T>();
+                        a.AsUInt32(), b.AsUInt32()).AsUInt16()).AsUInt64().As<ulong, T>();
+            }
+            if (Sse41.IsSupported)
+            {
+                Vector128<ulong> a = left.AsUInt64();
+                Vector128<ulong> b = left.AsUInt64();
```

Comment:
Same issue for Arm64:
```suggestion
                Vector128<ulong> a = left.AsUInt64();
                Vector128<ulong> b = right.AsUInt64();
```

Reply:
oops 😐 thanks